api/Element.json

```
@@ -1508,18 +1524,50 @@
             "opera_android": {
               "version_added": "11.5"
             },
-            "safari": {
-              "version_added": "6"
-            },
-            "safari_ios": {
-              "version_added": "5"
-            },
-            "samsunginternet_android": {
-              "version_added": "1.0"
-            },
-            "webview_android": {
-              "version_added": "≤37"
-            }
+            "safari": [
+              {
+                "version_added": "6.1"
+              },
+              {
+                "version_added": "6",
+                "version_removed": "6.1",
+                "notes": "Not supported for SVG elements.",
+                "partial_implementation": true
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": "7"
+              },
+              {
+                "version_added": "5",
+                "version_removed": "7",
+                "notes": "Not supported for SVG elements.",
+                "partial_implementation": true
+              }
+            ],
+            "samsunginternet_android": [
+              {
+                "version_added": "1.5"
+              },
+              {
+                "version_added": "1.0",
+                "version_removed": "1.5",
+                "notes": "Not supported for SVG elements.",
+                "partial_implementation": true
+              }
+            ],
+            "webview_android": [
+              {
+                "version_added": "≤37"
```

```suggestion
                "version_added": "4.4"
```

The change we're looking for was in WebKit 537.4. The last WebKit-based WebView was 4.2 using WebKit 534.30, then 4.4 was based on Chromium M30 which had the change.